src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -202,10 +205,20 @@ public boolean rightLowerLimitReached() {
 
     @Override
     public void periodic() {
-        if (leftLimitSwitch.isPressed() && leftLiftEncoder.getPosition() != 0) {
-            leftLiftEncoder.setPosition(0);
+        if (leftLimitSwitch.isPressed()) {
+            if (leftLimitSwitchBeenPressed == false) {
```

Comment:
I think we can just do switchPressed = true every time the switch is pressed, add this if statement adds lines of code but doesn’t change the outcome. In other words, I would put this back as it was and only add the setting of the Boolean var to the existing if statement